tests/login/login-admin.spec.ts

```diff
@@ -30,32 +30,32 @@ test('login page with admin privs', async ({ page }) => {
   expect(await btnLogin.count()).toEqual(1);
 
   // fill-up form - with admin privs
-  await inputEmail.click();
-  await inputEmail.fill(process.env.A_EMAIL || '');
-  await inputPassword.click();
-  await inputPassword.fill(process.env.A_PASSWORD || '');
-  await btnLogin.click();
+  // await inputEmail.click();
+  // await inputEmail.fill(process.env.A_EMAIL || '');
+  // await inputPassword.click();
+  // await inputPassword.fill(process.env.A_PASSWORD || '');
+  // await btnLogin.click();
 
```

Comment:
Q: Just curious why so many of these login form tests are commented. Not working yet?